docs/manual-cluster-repair.md

```diff
@@ -0,0 +1,193 @@
+```bash
+export ETCDCTL_API=3
+export ETCDCTL_CACERT=/etc/kubernetes/pki/etcd/ca.crt
+export ETCDCTL_CERT=/etc/kubernetes/pki/etcd/healthcheck-client.crt
+export ETCDCTL_KEY=/etc/kubernetes/pki/etcd/healthcheck-client.key
+```
+
+Retrieve currently known members list:
+```bash
+etcdctl member list
+```
+
+Example output:
+```bash
+2ce40012b4b4e4e6, started, ip-172-31-153-216.eu-west-3.compute.internal, https://172.31.153.216:2380, https://172.31.153.216:2379, false
+2e39cf93b81fb7ed, started, ip-172-31-153-246.eu-west-3.compute.internal, https://172.31.153.246:2380, https://172.31.153.246:2379, false
+6713c8f2e74fb553, started, ip-172-31-153-235.eu-west-3.compute.internal, https://172.31.153.235:2380, https://172.31.153.235:2379, false
+```
+
+By comparing the Nodes list with etcd members list (hostnames, IPs) we can find
+the ID of the missing etcd member (dead etcd member would be missing from the
+Nodes list, or will be in NotReady state).
+
+For example it's found that there is not control-plane Node with IP
+`172.31.153.235`. It means etcd member ID `6713c8f2e74fb553` is the one we are
+looking for to remove.
+
+To remove dead etcd member:
+```bash
+etcdctl member remove 6713c8f2e74fb553
+Member 6713c8f2e74fb553 removed from cluster 4ec111e0dee094c3
+```
+
+Now members list should display only 2 members.
+```bash
+etcdctl member list
+```
+
+Example output:
+```bash
+2ce40012b4b4e4e6, started, ip-172-31-153-216.eu-west-3.compute.internal, https://172.31.153.216:2380, https://172.31.153.216:2379, false
+2e39cf93b81fb7ed, started, ip-172-31-153-246.eu-west-3.compute.internal, https://172.31.153.246:2380, https://172.31.153.246:2379, false
+```
+
+Exit the shell in etcd pod.
+
+## Create a fresh VM replacement
+Assuming you've used Terraform to provision your cloud infrastructure, use
+`terraform apply` to restore cloud infrastructure to the declared state, i.e. 3
+control-plane instances for HA clusters.
+
+From your local machine:
+```bash
+terraform apply
+```
+
+The result should be: 3 running control-plane VM instances. Two existing and currently members
+of the cluster, and the fresh one which will be joined to the cluster as
+replacement for failed VM.
+
+## Join new VM to the cluster as control-plane node
+
+### What is Leader instance
+In KubeOne the Leader instance is the first instance from the `Hosts` in KubeOne
+config file (or `kubeone_hosts` in Terraform output) list. This instance will be
+used to "init" the cluster initially (generate PKI, start the first etcd
+instance, launch different kubernetes control-plane components).
+
+By default, the first `Host` instance defined in KubeOne config (or
+`kubeone_hosts` in Terraform output) will be a Leader Host.
+
+It's possible to choose which instance will be a Leader using config or
+Terraform output. Please keep in mind, there can be only 1 Leader Host.
+
+#### WARNING: It is super important to appoint one of the existing and healthy control-plane nodes as a Leader.
```

Comment:
```suggestion
**WARNING: It is super important to appoint one of the existing and healthy control-plane nodes as a Leader.**
```

You will get the same effect if you use bold. Using bold is a more correct way to do this, especially, if we start generating TOC someday.